src/Algebra/Graph/Undirected.hs

```diff
@@ -701,13 +701,11 @@ toSymmetricRelation = foldg SR.empty SR.vertex SR.overlay SR.connect
 {-# INLINE toSymmetricRelation #-}
 
 -- | Complement of a graph.
--- Complexity: /O(m^2+n)/ time, /O(m+n)/ memory where
+-- Complexity: /O(m^2)/ time, /O(m^2)/ memory where
 --
 -- @
 -- complement 'empty'           == 'empty'
 -- complement ('vertex' x)      == ('vertex' x)
--- complement ('edge' x y)      == ('vertices' [x, y])
--- complement ('star' x [y, z]) == ('overlay' ('vertex' x) ('edge' y z))
```

Comment:
These were rather helpful examples, even if incorrect :) Perhaps, we should use fixed values `1`, `2` and `3` both here and in the tests.

Reply:
I'll use them with fixed values, highlighting the gotcha for x == y